ndsc

```diff
@@ -98,9 +98,10 @@ def format_transaction_row(trn, width):
 
     def truncate_column(text, maxlen):
         if len(text) <= maxlen:
-            return pad(text, maxlen)
+            return str(pad(text, maxlen))
         else:
-            return text[0:maxlen-1] + '…'
+            # python 2 can't concatenate unicode and str-containing-unicode
+            return str(text[0:maxlen-1]) + '…'
```

Comment:
I'm not 100% sure I understand what's going on here.  Looking at the code and the comment, it seems like `'...'` is str-containing-unicode for python2, and `text[0:maxlen-1]` is unicode, so we have to force it to string in order to get the concatenation to work?  (This also has the side effect of making `truncate_column()` always return a str.)  But how would `text[0:maxlen-1]` end up being unicode?

Reply:
text[0:maxlen-1] is unicode because of the changes I made to python-discuss. In python3 the str does nothing, and in python2 I believe it was a str before I changed anything in python-discuss, so I don't think making it always return a str changes anything.